### PR TITLE
 FALCON-753 , FALCON-754. Correcting the authentication check in SLA and authenticating right user in Entity sla.

### PR DESCRIPTION
**Pull request by sandeepSamudrala**

### CODE REVIEW

**Comment by pallavi-rao** [comment]
@sandeepSamudrala were you able to do any dev-testing for these bugs? If not, we should set the expectations of QA accordingly.

**Comment by pallavi-rao** [comment]
Apart from the minor nit, +1.

**Comment by sandeepSamudrala** [comment]
@pallavi-rao : Made changes as discussed.

File: prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java [hunk 1]
```diff
@@ -142,17 +143,8 @@ private void startEntityMonitoring(Entity entity, boolean isEntityUpdated) throw
...
-                            if (isEntityRunning(entity)) {
-                                MONITORING_JDBC_STATE_STORE.putMonitoredEntity(process.getName(),
-                                        EntityType.PROCESS.toString(), new Date(now().getTime() + MINUTE_DELAY));
-                                List<Date> instances = EntityUtil.getEntityInstanceTimesInBetween(entity, cluster,
-                                        getStartTime(entity, cluster), now());
-                                addPendingInstances(entity.getEntityType().name().toLowerCase(), entity, cluster,
-                                        instances);
-                            } else {
-                                MONITORING_JDBC_STATE_STORE.putMonitoredEntity(process.getName(),
+                            MONITORING_JDBC_STATE_STORE.putMonitoredEntity(process.getName(),
```
Comment (sandeepSamudrala) [inline prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java]: The same changes were done to feed too.

**Pull request merged**